android_unpinner/__main__.py

```diff
@@ -227,9 +228,25 @@ def _force(ctx, param, val):
 )
 
 
+def _listen(ctx, param, val):
+    global gadget_config_file
+    gadget_config_file = gadget_config_file_listen
+
+
+listen_option = click.option(
+    "-l",
+    "--listen",
+    help="Specify frida-gadget's interaction mode to listening",
```

Comment:
Can we make the help text a bit more clearer here? Maybe something like this?

```suggestion
    help="Configure the Frida gadget to expose a server instead of running unpinning scripts",
```

Reply:
Updated the PR accordingly :)